proseco/core.py

```diff
@@ -996,6 +1003,28 @@ def null_logger(*args, **kwargs):
 
             return null_logger
 
+    def set_agasc_supp_version(self):
+        """
+        Get the versions of the agasc supplement used for the star data for catalog
```

Comment:
This doc string is not quite reflective of what the method is actually doing. There is one version and one last update time.